database-handler.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+class DatabaseHandler
+{
+    private $table_name;
+
+    public function init()
+    {
+        global $wpdb;
+
+        $this->table_name = $wpdb->prefix . "lightning_publisher_payments";
+
+        $charset_collate = $wpdb->get_charset_collate();
+
+        $sql = "CREATE TABLE $this->table_name (
+        id bigint(20) NOT NULL AUTO_INCREMENT,
+        post_id bigint(20) NOT NULL,
+        payment_hash text NOT NULL,
+        payment_request text NOT NULL,
+        amount_in_satoshi int(10) DEFAULT 0.0 NOT NULL,
+        exchange_rate int(10) DEFAULT 0.0 NOT NULL,
+        exchange_currency varchar(10) DEFAULT NULL,
+        created_at datetime DEFAULT CURRENT_TIMESTAMP NOT NULL,
+        settled_at datetime DEFAULT CURRENT_TIMESTAMP NOT NULL,
+        state tinytext NOT NULL,
+        PRIMARY KEY  (id)
+        ) $charset_collate;";
+
+        require_once(ABSPATH . 'wp-admin/includes/upgrade.php');
+        dbDelta($sql);
+    }
+
+    public function find()
```

Comment:
where is this find function used?

Reply:
aah, it's useless for now. I changed my mind about it's usage. Let me remove it